sdk/search/search-documents/test/public/node/searchClient.spec.ts

```diff
@@ -282,3 +283,57 @@ describe("SearchClient", function(this: Suite) {
     assert.equal(searchResults.count, 1);
   });
 });
+
+describe.only("SearchClient-local", () => {
```

Comment:
I think the analyze step is failing because of the `describe.only` here - now that we have the tests passing, we can remove the `.only` so all tests will run